platform/windows/os_windows.cpp

```diff
@@ -295,16 +302,19 @@ OS::TimeZoneInfo OS_Windows::get_time_zone_info() const {
 		daylight = true;
 	}
 
+	// Daylight Bias needs to be added to the bias if DST is in effect, or else it will not properly update.
 	TimeZoneInfo ret;
 	if (daylight) {
 		ret.name = info.DaylightName;
+		ret.bias = info.Bias+info.DaylightBias;
 	} else {
 		ret.name = info.StandardName;
+		ret.bias = info.Bias;
```

Comment:
```suggestion
		ret.bias = info.Bias + info.StandardBias;
```

Should we add it as well? AFAIK, it's currently zero for all zones, but it might change.

Reply:
In my limited testing this doesn't seem to introduce any errors so sounds good.

Reply:
Should this be done if `daylight` is true as well?

Reply:
No, `StandardBias` seems to be for non-DST only.